### DISCUSSION TITLE
[Feature/CSS Request] Dynamically hide Firefox's tabs of other folders/trees

### DISCUSSION THREAD

**Discussion by couleurm** [piroor/treestyletab#3553] [Ideas]
I have tons of folders and would like that all of the Firefox's own vanilla horizontal tabs are filtered depending on what currently tab you have opened contains

**Comment by irvinm** [comment]
TST is basically just a virtual representation of Firefox's native tabs.  If you wanted to hide the native tabs, you can easily do that by calling tabs.hide() from an addon [[https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/hide](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/hide)] but that would also hide it from TST.

I believe TST honors the visibility of a tab and if it is hidden from the horizontal tabs bar, it will be hidden from the sidebar as well.
